src/components/block.ts

```diff
@@ -28,6 +28,18 @@ import DeleteTune from './block-tunes/block-tune-delete';
 import MoveDownTune from './block-tunes/block-tune-move-down';
 import SelectionUtils from './selection';
 
+/**
+ * Available Block Tool API methods
+ */
+export enum BlockToolAPI {
```

Comment:
maybe it would better to place in `tools.ts` near `INTERNAL_SETTINGS` and `USER_SETTINGS` ? and use same structure

Reply:
I think it is better to left it here because in this case `Block` class be encapsulated as it is now